core/wiki/paletteswitcher.tid

```diff
@@ -2,15 +2,18 @@ title: $:/snippets/paletteswitcher
 
```

Comment:
Can you add indentation, so it's easier to read

Reply:
Good point will do